Exercises/promise-async-await/promise-async-await.js

```diff
@@ -1,7 +1,54 @@
+/*eslint-disable no-console*/
+//Tasks:
+//1. Make function which will return promise which will resolve in passed to function amount of milliseconds.
+const util = require('util');
+
+function callFunction(func, ms) {
```

Comment:
Currently you just return the function from your promise. The function should be called after timeout, and it's result should be returned. The point is to create promisified setTimeout